examples/thoughts/ext/Tag.css

```diff
@@ -0,0 +1,24 @@
+.tag {
+  margin: 0 10px;
+  text-decoration: none;
+  cursor: text;
+}
+
+.tag.deletable:hover {
+  text-decoration: line-through;
+  cursor: pointer;
+}
+
+.tag.link {
+  padding: 5px;
+  text-decoration: none;
```

Comment:
Is this line redundant since we already have `text-decoration: none` for `.tag`?

Reply:
Yes it is, removed.